src/controllers/itemdetailpage.js

```diff
@@ -519,11 +520,19 @@ define(["loading", "appRouter", "layoutManager", "userSettings", "connectionMana
             itemBackdropElement.style.backgroundImage = "";
         }
 
+        if ("Person" === item.Type) {
+            itemBackdropElement.classList.add("personBackdropimg-hideondesktop");
+            itemBackdropElement.classList.add("personBackdrop");
```

Comment:
why cannot you merge two CSS rules into just `personBackdrop`?

Reply:
that will do thanks https://github.com/jellyfin/jellyfin-web/pull/694/commits/89b28d76f58af92c3e42c717b6f34ae593544b30